paasta_tools/contrib/service_shard_update.py

```diff
@@ -201,11 +208,17 @@ def main(args):
         working_dir=args.local_dir or "/nail/tmp",
         do_clone=args.local_dir is None,
     )
+    deploy_environments = DEPLOY_MAPPINGS
+    if args.environment is not None:
+        deploy_environments = {
+            k: v for k, v in DEPLOY_MAPPINGS.items() if k == args.environment
+        }
```

Comment:
isn't this essentially
```suggestion
    deploy_environments = {args.environment: DEPLOY_MAPPINGS[args.environment]} if args.environment else DEPLOY_MAPPINGS
```
?

Reply:
ooo nice, that looks great!